src/app/movies/[id]/page.tsx

```diff
@@ -42,37 +42,42 @@ export default async function Movie({ params }: { params: { id: string } }) {
       <MovieInfo movie={movie} />
 
       <div className="section-padding">
-        <Section
-          header={
-            <div className="header-default-style">
-              <h3>Cast</h3>
-              <Link
-                href={`${params.id}/cast`}
-                aria-label="See all cast members"
-              >
-                See all
-              </Link>
-            </div>
-          }
-        >
-          <PersonList persons={movie.cast.slice(0, 6)} />
-        </Section>
+        {movie.cast.length > 0 && (
```

Comment:
Tarviiko näitä piilottaa, kun tuolla on tuo `No cast/crew to show` teksti jo ?

Reply:
Joo voi nämä piilottaa. En tiedä miettisikö käyttäjä sitten missä crew ja cast on kun niitä ei näy leffan sivulla.

Reply:
Sitä määkin aloin miettiä että onko kuitenkin käyttäjän näkökulmasta parempi että ne näkyy siellä ja on vaan että tätä tietoa ei oo nyt saatavilla. 🤔 Ehkä se vois olla ihan hyvä, pyörittelen tota vielä vähän ku pääsen cypress-ongelmista eroon.